test/common_utils.py

```diff
@@ -550,19 +559,26 @@ def fn(shape, dtype, device):
             )
         )
 
-        image_tensor = convert_dtype_image_tensor(to_image_tensor(image_pil).to(device=device), dtype=dtype)
+        image_tensor = to_image_tensor(image_pil)
+        if memory_format == torch.contiguous_format:
+            image_tensor = image_tensor.to(device=device, memory_format=memory_format, copy=True)
+        else:
+            image_tensor = image_tensor.to(device=device)
+        image_tensor = convert_dtype_image_tensor(image_tensor, dtype=dtype)
+        assert image_tensor[None].is_contiguous(memory_format=memory_format)
```

Comment:
do we really want this assert here, or was it just a sanity check during development?

Reply:
Yeah, a sanity check to ensure that input is the one we want. I can remove it you think it is unnecessary.